bin/.eslintrc.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
I created this one in the `bin` directory to deactivate the `no-console` rule just here.

I propose not to deactivate it everywhere, because usage of `console.log` can be signs of forgotten test code. But if you prefer I can move it to the general configuration (but deactivate it in tests)

Reply:
It was a great time with you